src/sass/page-template-variables.scss

```diff
@@ -297,7 +297,7 @@
     // CLICK TO EXPAND
     --theme-click-to-expand-copy-height-before-fade-starts: 100px;
     --theme-click-to-expand-copy-total-height: 450px;
-    --theme-click-to-expand-cta-image-url: url("https://aaf1a18515da0e792f78-c27fdabe952dfc357fe25ebf5c8897ee.ssl.cf5.rackcdn.com/1839/btn+more.png?v=1614305726000");
+    --theme-click-to-expand-cta-image-url: url("https://aaf1a18515da0e792f78-c27fdabe952dfc357fe25ebf5c8897ee.ssl.cf5.rackcdn.com/1839/more-btn.png?v=1614773932000");
```

Comment:
Did you delete the old (low res) image asset off their account?

Reply:
Yes